src/main/resources/swagger.api/inn-reach.yaml

```diff
@@ -789,6 +789,19 @@ paths:
       parameters:
         - $ref: '#/components/parameters/centralServerId'
         - $ref: '#/components/parameters/inventoryInstanceId'
+  /inn-reach/contribution/start-initial-contribution:
```

Comment:
do you mind changing path to `/central-servers/{centralServerId}/contributions/start-initial-contribution` ? This way it will be aligned with endpoints for contribution job status and statistics